source/learn.html.erb

```diff
@@ -116,25 +116,52 @@ responsive: true
     <section>
       <h2 id="faq">Frequently Asked Questions</h2>
 
-      <h3 id="faq-future-proof">How do I future-proof my Ember.js application?</h3>
+      <h3 id="faq-glimmer">What is Glimmer? Do I need to learn it too?</h3>
 
       <p>
-        Ember.js has a strong emphasis on <a href="https://speakerdeck.com/wycats/stability-without-stagnation-lessons-learned-from-shipping-ember-dot-js">Stability without Stagnation</a>, which means that framework developers take great care when designing new functionality to make sure it fits with the existing functionalities, and provides a path forward for users to upgrade without being subjected to churn.
+        Glimmer is a component library used in Ember. However, in order to build Ember apps, you don't need to know anything about Glimmer, other than it's super fast and that the contributors to the core codebase love to geek out about it. It's all behind the scenes. If you're curious to know how the Glimmer project fits into the roadmap for Ember's development, you can <a href="https://discuss.emberjs.com/t/readers-questions-will-breaking-up-ember-remove-the-need-for-glimmer-js/14325">learn more here</a>.
```

Comment:
I think Glimmer is the rendering engine in Ember, not a component library.

Reply:
I had rendering engine at first and then I googled Glimmer and changed it. I'll put it back.